### ISSUE TITLE
test validated methods without meteor / publish to npm

### ISSUE THREAD

**Issue by macrozone** [meteor/validated-method#58]
We use validatedMethods in a mantra-app where we want to unit-test the methods. The test runs just wich mocha (without meteor), so it has no idea of meteor-packages.

By looking at the code, this package has little depencencies on meteor-stuff (e.g. Match), so it should be easy to create a npm-version of it. 

Thoughts?

**Comment by stubailo** [comment]
Good idea! I'd happily help you publish to npm if you can do the implementation.

**Comment by abernix** [comment]
@macrozone Thoughts on making an npm-based implementation of this? This would be a valuable Meteor contribution, and we could (officially) publish it as `@meteorjs/validated-method`!
